src/niscope/system_tests/test_system_niscope.py

```diff
@@ -8,6 +8,11 @@
 import tempfile
 
 
+instruments = ['FakeDevice1', 'FakeDevice2']
+# TODO(sbethur): Use `get_channel_names` when #1402 is fixed
```

Comment:
Nothing is stopping us from fixing this, right?

Reply:
That's right. We just need to prioritize it.